src/gpu/nvidia/cudnn_lrn.cpp

```diff
@@ -20,6 +20,8 @@
 #include "gpu/nvidia/sycl_cuda_stream.hpp"
 #include "sycl/sycl_buffer_memory_storage.hpp"
 
+#include "sycl_cuda_helper.hpp"
```

Comment:
Hi! Please check up with comments from https://github.com/oneapi-src/oneDNN/pull/1258.

Could you also consider apply the patch on top of the branch from that PR to have only changes related to new functionality, please?

Reply:
I have updated based upon the comments in #1258.